### PR TITLE
Table setup sf

### PR DESCRIPTION
**Pull request by StephanieFriend**
### Type of change made:
- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling - no new features
### Why is this change required? What problem does it solve?
Adds a timestamp feature to all the tables with the math to add the created_at and auction_length columns in the items table to make the auction_end column
### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
ALL THE CHALLENGES
NOTE: The default setting for auction_length in the items is currently not working. Needs to be set manually and HAS to be in minutes. Will work on finding a solution.

### CODE REVIEW

File: application/controllers/bid_details.py [hunk 1]
```diff
@@ -68,11 +69,12 @@
 )
 bid_detail_post_parser.add_argument(
     'created_at',
-    type=datetime,
+    type=int,
     required=False,
```
Comment (reid-andrew) [inline application/controllers/bid_details.py:73]: Interesting. But this makes perfect sense & should be something the FE can work with.

**Review by reid-andrew — APPROVED** [review]

File: application/controllers/bids.py [hunk 2]
```diff
@@ -83,6 +84,8 @@ def post(self):
             abort(404, description='That item or user does not exist')
         else:
             bid = Bid(**args)
+            dt = trunc(datetime.now().timestamp())
```
Comment (kmcgrevey) [inline application/controllers/bids.py:87]: curious about this `trunc` function . . . `trancate`?

File: application/controllers/bids.py [hunk 3]
```diff
@@ -13,7 +14,7 @@
     'user_id': fields.Integer,
     'amount': fields.Float,
     'winner': fields.Boolean,
-    'created_at': fields.String
+    'created_at': fields.Integer,
```
Comment (kmcgrevey) [inline application/controllers/bids.py:17]: are you using Integer now because the date is logged in seconds instead of the traditional `datetime` syntax?

**Review by kmcgrevey — APPROVED** [review]

File: application/controllers/bids.py (hunk 2 above)
Comment (StephanieFriend) [reply application/controllers/bids.py:87]: This function removes any trailing decimals as a timestamp will give you seconds and milliseconds. We only want to return the seconds so I went ahead and chopped off the milliseconds.

File: application/controllers/bids.py (hunk 3 above)
Comment (StephanieFriend) [reply application/controllers/bids.py:17]: We are actually using traditional timestamps. Which is typically utilized universally. We just haven't had the opportunity to work with them yet. A time stamp is the seconds from Jan 1st 1970. There is a library in JS that will run that timestamp and be more friendly to the FE to help them do what they need with a countdown.

File: application/controllers/bid_details.py (hunk 1 above)
Comment (StephanieFriend) [reply application/controllers/bid_details.py:73]: Agreed. JS has a library that will interpret the timestamp much simpler than if we try to do any string or datetime work on the BE.

**Pull request merged**